File: Desenvolvedor de Apps/Angular/AngularAndDotNet/AngularDotNet.API/People.cs
using System.ComponentModel.DataAnnotations;

namespace AngularDotNet.API
{
    public class People
    {
        [Key]
        public Guid idPeople { get; set; }
        public string namePeople { get; set; } = string.Empty;
        public string lastNamePeople { get; set; } = string.Empty;
        public int agePeople { get; set; }
        public string professionPeople { get; set; } = string.Empty;

    }
}

File: Desenvolvedor de Apps/Angular/AngularAndDotNet/AngularDotNet.UI/src/app/People.ts
export class People {
    peopleId: string;
    namePeople: string;
    lastNamePeople: string;
    agePeople: number;
    professionPeople: string;

    constructor (peopleId: string, namePeople:string, lastNumberPeople:string, agePeople: number, professionPeople: string)
    {
        this.peopleId = peopleId,
        this.namePeople = namePeople,
        this.lastNamePeople = lastNumberPeople,
        this.agePeople = agePeople,
        this.professionPeople = professionPeople
    }
}